app/helpers/workflow_helper.rb

```diff
@@ -4,7 +4,7 @@ def show_workflow_grid?
   end
 
   def render_workflow_grid
-    workflow_data = facet_tree('wf_wps')['wf_wps_ssim']
+    workflow_data = params[:all] ? facet_tree : facet_tree('wf_wps')['wf_wps_ssim']
```

Comment:
The vanilla `report_workflow_grid_path` route should do this already. Is it not?